source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -16,35 +16,60 @@ reviewed: "2022-08-11"
 
 ## Create the CMS Backend
 
-You can create a decoupled site using a Pantheon starter kit. Choose from either a WordPress or Drupal CMS and connect it to the frontend application to create a site. The following configurations exist for the starter kit templates:
+You can create a Front-End Site by configuring your Dashboard upstream or by using a Pantheon starter kit. Choose either a WordPress or Drupal CMS and connect it to the frontend application to create a site. The following configurations exist for the starter kit templates:
 
 * [Drupal and Next.js](https://github.com/pantheon-systems/decoupled-drupal-recommended) - This configuration supports server-side rendering. 
 * [WordPress and Gatsby](https://github.com/pantheon-systems/decoupled-wordpress-recommended) - This configuration supports static site generation. 
 
 
 ## Backend Installation and Configuration
 
+Choose from the following approaches to configure your backend:
+
+* Build Tools:
+  * Use this option if testing is important to your team's workflow. With Build Tools you will not have to manually push changes to your code repository.
```

Comment:
Maybe change to 'you will not have to manually push changes to your Pantheon code repository.'

They will have to push code somewhere, it just happens to be their external repository.